lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -0,0 +1,133 @@
+defmodule ExUnit.CaptureLog do
+  @moduledoc ~S"""
+  Functionality to capture logs for testing.
+
+  ## Examples
+
+      defmodule AssertionTest do
+        use ExUnit.Case
+
+        import ExUnit.CaptureLog
+
+        test :example do
+          assert capture_log(fn ->
+            Logger.error "log msg"
+          end) =~ "log msg"
+        end
+
+        test "check multiple captures concurrently" do
+          fun = fn ->
+            for msg <- ["hello", "hi"] do
+              assert capture_log(fn -> Logger.error msg end) =~ msg
+            end
+            Logger.debug "testing"
+          end
+          assert capture_log(fun) =~ "testing"
+        end
+      end
+
+  """
+
+  @doc """
+  Captures Logger messages generated when evaluating `fun`.
+
+  Returns the binary which is the captured output.
+
+  By default, `capture_log` replaces the `group_leader` (`:stdio`)
+  for the current process and captures any log messages sent by
+  processes wth the replacement `group leader`. However, the
+  capturing of log messages with a `group_leader` of any other
+  device is also possible by giving the device explicitly as an
+  argument. It is also possible to capture all log messages by
+  giving `nil` for the device.
+
+  Note that when capturing something other than `:stdio`, log
+  messages from other tests or processes might be captured. However
+  it is safe to use async true if this can be handled. Multiple
+  captures (and other backends) can handle log messages at the same
+  time. For example the `:console` backend will still log messages
+  to the console.
+
+  It is possible to capture all log messages by giving `nil` for
+  the device.
+
+  It is possible to configure the level to capture with `:level`,
+  which will change the current Logger level for the duration of the
+  capture. If the log level is changed while capturing, for example
+  by another `capture_log/3`, the behaviour is undetermined.
+  Therefore tests should be async false when setting a capturing
+  level. The default level is `nil`, which will capture all messages
+  at the current Logger level. It is safe to use with async true when
+  the level is `nil` because it does not change the current Logger
+  level.
+
+  The format, metadata and colors can be configured with `:format`,
```

Comment:
Yes, the question is: if I configure console, will it automatically configure the capture, or do I need to do both?